src/examples/files.jl

```diff
@@ -52,3 +52,12 @@ dirresponse(f) =
             div(".box", table([tr(td(".file", filelink(f, x)),
                                   td(".size", string(filesize(joinpath(f, x)))))
                                for x in ["..", readdir(f)...]]))))
+
+const ASSETS_DIR = "assets"
+function packagefiles(dirs=true)
+    absdir(req) = Pkg.dir(req[:params][:pkg], ASSETS_DIR)
+    branch(req -> Mux.validpath(absdir(req), joinpath(req[:path]...), dirs=dirs),
+           req -> Mux.fresp(joinpath(absdir(req), req[:path]...)))
```

Comment:
Are the `Mux.` prefixes necessary here? Also, does this guard against `/pkg/MyPkg/../src` type hacks? (it probably does but I've just forgotten what all this code does :)